frontend/src/pages/CreatePost/CreatePost.module.css

```diff
@@ -8,20 +8,60 @@
   left: calc(50% - 300px);
   background-color: white;
   padding: 10px 10px 10px 10px;
+  font-family: 'Roboto', sans-serif !important;
+}
+
+.heading {
+  margin-bottom: 30px !important;
+}
+
+.errorMessage {
+  position: absolute;
+  left: 15px !important;
+  bottom: 5px !important;
+  color: red;
 }
 
 .bodyText {
   margin-left: 5px;
   margin-right: 5px;
+  background-color: #e4e9ef;
+  margin-top: 0 !important;
+  padding: 2px 10px 2px 10px !important;
 }
 
 .titleText {
   margin-left: 5px;
   margin-right: 5px;
+  background-color: #e4e9ef;
+  margin-top: 0 !important;
+  margin-bottom: 20px !important;
+  padding: 2px 10px 2px 10px !important;
 }
 
 .modalButtons {
   position: relative;
   margin-left: auto;
   margin-right: 0;
 }
+
+.cancelButton {
+  background-color: #a4a9af !important;
+  font-size: large;
+  color: white !important;
+  margin-right: 10px !important;
+  border-radius: 0 !important;
+  height: 35px;
+  padding: auto !important;
+  font-family: 'Roboto', sans-serif !important;
+
+}
+
+.submitButton {
+  background-color: #0079d3 !important;
+  color: white !important;
+  border-radius: 0 !important;
```

Comment:
Having all the `!important` overrides seem a bit concerning. I'm guessing this is due to an issue of ordering when the CSS files are placed on the page and how it conflicts with Material-UI's internal styling system?

Would be good to see how easy it is to fix this. If it requires large refactor of CSS to fix it, then I can say to leave it as now and it can be cleaned up later in the future™ .

Reply:
Yeah, the spacing would be the only nitpick I have. Also, the button colour looks slightly off. could we change the hex value to #107ccd? My mistake in the colour palette.

Reply:
I just changed the margins (they are now more evenly spaced). And also changed the button color to 107ccd. Removed some of the css lines with !important.

Reply:
Awesome! Lgtm

Reply:
Also attempted to change the font, and it is just not working for some reason. If someone could take a stab at it, would be much appreciated.